konveyor/pelorus
# Add troubleshooting docs

## Describe the behavior changes introduced in this PR

Addresses [the remaining todo items listed in #549](https://github.com/konveyor/pelorus/issues/549#issuecomment-1219820767), as well as some feedback from

- [ ] "how to ready your data for pelorus" docs section
- [ ] move troubleshooting from demo to top-level doc section

## Linked Issues?

resolves #549 

## Testing Instructions

Run a test as in #615 (with human-readable output) and verify that suggestions and links to the troubleshooting URL are present.
